enterprise/releases/2023-03-0.mdx

```diff
@@ -0,0 +1,122 @@
+  - Enforce maximum time for each snapshot collection using deadlines
+    - This will help ensure consistent reporting intervals for collectors with a lot of monitored servers, and avoid "noisy neighbor" problems during reporting
+  - Amazon RDS / Aurora
+    - Avoid DescribeDBInstances calls for log downloads for RDS instances
+    - Auto-detection of Aurora instances from the cluster hostname
+    - Multiple improvements to Log Insights integration
+  - Additional statistics tracking for autovacuum and TXID consumption rates
+  - Add Postgres 15 support
+  - See [full collector changelog](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md)
+- Better account management
+  - Allow calling the membership delete action via the GraphQL API
+  - SAML integration: Easier migration of existing users when turning on SAML for the first time
+  - SAML integration: Option to add identity provider metadata manually (instead of via URL)
+- Improved API key management
+  - Ability to create collector and read-only keys directly in the pganalyze UI
+  - Ability to revoke API keys (e.g. useful for key rotation)
+  - API key values are hidden by default when viewing the API keys page
+  - Increased random bytes for API keys for additional protection
+  - New API keys now start with a "pga" prefix, to make it easier to identify pganalyze API keys
+  - API key usage is now tracked in the `audit_logs` table (at most once per day per client IP)
+  - API key creation and revocation is now tracked in the `audit_logs` table
+- VACUUM monitoring improvements
+  - New per-table graph that shows the age of the oldest unfrozen XID on a table (to help understand anti-wraparound vacuums better)
+  - Extend VACUUM data retention time to 35 days (from 7 days)
+  - Additionally this release includes preparation for new autovacuum metrics and tuning recommendations, coming in a future Enterprise Server release
+- Other improvements
+  - Show Amazon RDS / Aurora instance tags on servers in pganalyze
+    - For instance with tags applied in AWS (e.g. to indicate which application team owns the instance),
+      pganalyze will now automatically show these tags in the server overview list
+  - Add `DISABLE_DIRECT_SIGNUP` setting to prevent uninvited users from registering a user account
+    - In case you are hosting your pganalyze instance on a publicly accessible host, we recommend enabling this new setting as an extra measure to prevent unexpected user registrations
+    - Note that this setting is not compatible with the `DEFAULT_ORG_ROLE` setting (which when enabled automatically adds new users to the initial organization in pganalyze)
+  - Add support for AWS IMDSv2 instance metadata protocol to retrieve credentials for accessing object storage
+  - Table columns page: Show NULL % and avg. width (requires [pganalyze.get_column_stats](/docs/install/amazon_rds/01_configure_rds_instance) helper)
```

Comment:
I think this is pointing to the wrong page?
```suggestion
  - Table columns page: Show NULL % and avg. width (requires [pganalyze.get_column_stats](/docs/install/amazon_rds/02_create_monitoring_user) helper)
```
We should probably have standalone troubleshooting pages for these, but that's a separate problem.